src/OptOut.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\OptOut.
+ */
+
+namespace Netzstrategen\CoreStandards;
+
+class OptOut {
+
+  /**
+   * Default Google Analytics opt out script.
+   *
+   * @var string
+   */
+  private static $opt_out_script = <<<CODE
+<script>
+// Disable tracking if the opt-out cookie exists.
```

Comment:
This script should be in an asset JS file and the GA IDs should be read from JS settings variables of the global document scope.

In case you intentionally inlined the script for frontend loading performance reasons, then you can still do that by including the compiled script from the dist folder instead.